Icon-Legend.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+<!-- 
+    website Refrence for icons 
+    https://www.wta.org/go-hiking/hikes/surprise-lake-1 -->
+
+    <div class="legend">
+
+
+    <table >
+        <col  style="background-color:rgb(0, 255, 213)">
+        <col style="background-color:rgb(172, 158, 255)">
+      </colgroup>
+        <tr>
+            <th> Icon  </th>
+            <th> Description </th>
+
+        </tr>
+
+        <tr>
+            <th > <img src="icon-images/Parking.png" alt="limited-parking">  </th>
```

Comment:
Should this be a <td> tag because it is a data cell under the "Icon" header for the table?

Reply:
Why not make two tr's?  I thought you were supposed have just one header for the table.